### PR TITLE
COST-2881: Purge trino files internal endpoint.

### PR DESCRIPTION
**Pull request by myersCody**
## Jira Ticket

[COST-2881](https://issues.redhat.com/browse/COST-2881)

## Description

This change will ...

## Testing

1. Checkout Branch
2. Restart Koku
3. Hit endpoint or launch shell
    1. You should see ...
4. Do more things...

## Notes

...

### CODE REVIEW

File: koku/masu/api/purge_trino_files.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+@api_view(http_method_names=["GET", "DELETE"])
+@permission_classes((AllowAny,))
+@renderer_classes(tuple(api_settings.DEFAULT_RENDERER_CLASSES))
+def purge_trino_files(request):
+    """
+    This endpoint deletes the parquet & csv files in s3.
+    Required Params:
+        provider_uuid - source_uuid
+        schema - account schema
+        bill_date - usually the start of the month example 08-12-2022
```
Comment (lcouzens) [inline koku/masu/api/purge_trino_files.py:39]: I feel like this should be yyyy-mm-dd

File: koku/masu/database/report_manifest_db_accessor.py [hunk 2]
```diff
@@ -251,3 +251,21 @@ def get_manifest_list_for_provider_and_date_range(self, provider_uuid, start_dat
             .filter(partition_date__gte=start_date, partition_date__lte=end_date)
         )
         return manifests
+
+    def bulk_delete_manifests(self, provider_uuid, manifest_id_list):
+        """
+        Deletes a specific manifest given manifest_id & provider_uui
```
Comment (lcouzens) [inline koku/masu/database/report_manifest_db_accessor.py:257]: provider_uuid

**Review by lcouzens — CHANGES_REQUESTED** [review]
We need to update the masu openapi spec + add unit tests.

**Comment by myersCody** [comment]
> We need to update the masu openapi spec + add unit tests.

Thanks for the early feedback, I went ahead and address these while I was working on the endpoint. I should be ready to move this out of draft later tonight after checking the smoke tests.

**Review by lcouzens — DISMISSED** [review]

**Comment by lcouzens** [comment]
Failing smoke tests change on each run, sometimes time-outs sometimes not on different tests. Either way the smoke tests really don't test anything in this pr directly so I think this is fine to move forward.

**Comment by myersCody** [comment]
Smoke test: https://ci.ext.devshift.net/job/project-koku-koku-pr-check/6330/

We are certain the one failure is unrelated to these changes.

**Review by lcouzens — APPROVED** [review]

**Pull request merged**